eng/Versions.props

```diff
@@ -112,8 +112,8 @@
     <MicrosoftVisualStudioSDKEmbedInteropTypesVersion>15.0.27</MicrosoftVisualStudioSDKEmbedInteropTypesVersion>
     <MicrosoftVisualStudioEditorVersion>$(VisualStudioEditorPackagesVersion)</MicrosoftVisualStudioEditorVersion>
     <MicrosoftVisualStudioGraphModelVersion>16.0.28226-alpha</MicrosoftVisualStudioGraphModelVersion>
-    <MicrosoftVisualStudioImageCatalogVersion>16.4.29315.20-pre</MicrosoftVisualStudioImageCatalogVersion>
-    <MicrosoftVisualStudioImagingVersion>16.4.29315.20-pre</MicrosoftVisualStudioImagingVersion>
+    <MicrosoftVisualStudioImageCatalogVersion>16.6.29925.50-pre</MicrosoftVisualStudioImageCatalogVersion>
```

Comment:
you should be able to use any non-prerelease version from https://dev.azure.com/azure-public/vside/_packaging?_a=package&feed=vssdk%40Local&package=Microsoft.VisualStudio.Shell.Framework&protocolType=NuGet&version=16.7.30021.50&view=overview

I guess you can pick anything greater than `16.6.29925.50-pre` which will give you the least grief. `16.6.30020.184` is the greatest version within `16.6` as of now.